src/cli/CommandLineIface.cpp

```diff
@@ -245,3 +254,48 @@ void CommandLineIface::outputTranslation(Translation output) {
     outstream_.flush();
     eventLoop_.exit(); // Unblock the main thread
 }
+
+int CommandLineIface::allowNativeMessagingClient(QStringList ids) {
+    if (ids.isEmpty()) {
+        qCritical().noquote() << "No client ids specified";
+        return 1;
+    }
+
+    auto clients = settings_.nativeMessagingClients();
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
```

Comment:
Can we unify the codepath here? Or is the OLD qt version way of doing things deprecated?

Reply:
`toSet()` has disappeared completely from the documentation so I don't expect it to work anymore. The canonical way of doing it is through the iterator initialisers but those are only available since 5.14 which is somehow not what Ubuntu 18.04 ships with.